.github/continuous-integration.sh

```diff
@@ -56,7 +60,13 @@ function check-ccache {
 }
 
 function run-build {
-    A2M_CCACHE="1" BUILD_FLAVOUR="Release" A2M_ASSERTS="1" run-step ./build
+    A2M_CMAKE_DEFS=('BUILD_FLAVOUR="Release"' 'A2M_ASSERTS="1"')
+
+    if [[ ! -z "${PERSIST_DIR}" ]]; then
+        A2M_CMAKE_DEFS+=('A2M_CCACHE="1"')
+    fi
+
+    "${A2M_CMAKE_DEFS[@]}" run-step ./build
```

Comment:
Sorry for that suggestion, we have to do `env "${A2M_CMAKE_DEFS[@]}" run-step ./build`for the environment variables to be set.